RetakesPlugin/Modules/Configs/RetakesConfigData.cs

```diff
@@ -19,4 +19,6 @@ public class RetakesConfigData
     public bool IsDebugMode { get; set; } = false;
     public bool ShouldForceEvenTeamsWhenPlayerCountIsMultipleOf10 { get; set; } = true;
     public bool EnableFallbackBombsiteAnnouncement { get; set; } = true;
+    public bool RemoveSpectators { get; set; } = true;
```

Comment:
Default this value to false, also as it's a boolean it needs to be prefixed with `Is` / `Has` / `Should` / `Enable`